client/src/components/J40footer.tsx

```diff
@@ -1,17 +1,56 @@
 import React from 'react';
-import { Footer } from '@trussworks/react-uswds';
-import {Link} from 'gatsby-plugin-intl';
-
-const footerLinks = [
-    <Link to="/">Home</Link>
-];
+import {Footer, Logo, FooterNav, Address} from '@trussworks/react-uswds';
+// @ts-ignore
+import whitehouseIcon from '../images/eop-seal.svg';
 
 const J40Footer = () => {
-    return (
-        <>
-        <Footer primary={[]} secondary={footerLinks} />
-        </>
-    );
+  return (
+    <>
+      <Footer
+        size="big"
+        primary={<></>}
+        secondary={<FooterNav
+          aria-label="Footer navigation"
+          size="big"   // fyi you leave this off and it silently fails... lovely
+          className={"nobreak"}
+          links={[
+            [
+              "Agency Partners",
+              <a href="https://www.epa.gov/">Environmental Protection Agency</a>,
```

Comment:
Another small point actually is link underlines - I just left a comment about this in the spec as well, but by default links have underlines and the spec links currently do not. 

Again I think the right thing is to leave as-is here, get default link styling behavior, and change our spec, but just wanted to mention this